extensions/kafka/pom.xml

```diff
@@ -64,6 +64,11 @@
             <artifactId>kafka-clients</artifactId>
             <version>${kafka.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Probably worth verifying that everything works with a clean project with HZ as an external library. I'm not sure how shading works for this module exactly.

Reply:
0d2d85a

Reply:
are users expected to add `org.apache.avro:avro` to their member class path in order to use avro schema? This might breaks with shading, because refereneces in our JAR to Avro lib will be shaded.
It also will be necessary in Viridian to add appropriate dependency. Maybe we should add is as a `compile` dependency?

Reply:
Avro is probably shaded in the avro-extension too, if we ensure versions are the same, theoretically someone shading both extensions should get one single com.hazelcast.org.apache.avro.* shaded package, used by both modules. But this needs to be verified.

Versions should already be matched project-wide, in theory. 

This scenario is probably worth checking: 
- Creating a simple maven project 
- Adding hazelcast-avro and hazelcast-kafka extensions (in addition to SQL)
- Setting up shading similar to ours e.g. maven-assembly-plugin
- Verifying that both Kafka and Avro connectors work correctly (think just INSERT-SELECT should be enough).

Reply:
`hazelcast-jet-avro` seems to be the only module including Avro dep not as provided/test scope. In order to use Kafka with Avro users would need to have `hazelcast-jet-avro`  and `hazelcast-jet-kafka` on their classpath. This seems to make sense. These extensions are included in the full `distribution`, also in EE which is based on OS distribution.